pygmt/src/inset.py

```diff
@@ -0,0 +1,106 @@
+"""
+inset - Create inset figures.
+"""
+import contextlib
+
+from pygmt.clib import Session
+from pygmt.helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(D="position", F="box", M="margin", V="verbose")
```

Comment:
Could you please also add an alias for **-N**? I think `no_clip` is a good alias, just like `Figure.plot()` and `Figure.text()`.

Reply:
```suggestion
@use_alias(D="position", F="box", M="margin", N="no_clip", V="verbose")
```